src/app/containers/ArticleMain/index.test.jsx

```diff
@@ -65,3 +66,48 @@ describe('ArticleMain', () => {
     </Context>,
   );
 });
+
+describe('processArticleData', () => {
```

Comment:
This tests should be in `processArticleData.test.js`, or ideally something specifically named like `containers/ArticleMain/metadata/index.test.js`